CMakeLists.txt

```diff
@@ -157,7 +158,7 @@ IF(Boost_FOUND)
 	SET(BOOST_PARAMETER_MAX_ARITY 7)
 	ADD_DEFINITIONS(-DBOOST_PARAMETER_MAX_ARITY=${BOOST_PARAMETER_MAX_ARITY})
 ELSE(Boost_FOUND)
-	MESSAGE(FATAL_ERROR "Boost 1.46 or newer is needed to build OpenCog!")
+	MESSAGE(FATAL_ERROR "Boost 1.46 or newer (package libboost-all-dev) is needed to build OpenCog!")
```

Comment:
Clarifies error message to specify that libboost-all-dev is to be used. Took me awhile to figure that out myself